### PR TITLE
ES build.gradle plugin dependency change for patch release.

### PR DESCRIPTION
**Pull request by peterzhuamazon**
This PR is to resolve the ES build.gradle plugin dependency change for patch release.

### Test Cases: ###
~~https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/203332853~~
https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/204457722

Please check the logs to see that dependencies are correctly set up now.

Thanks.

### CODE REVIEW

File: bin/plugins.json [hunk 1]
```diff
@@ -1,60 +1,231 @@
...
-    {"s3": "opendistro-security/opendistro-security", "git": "opendistro-for-elasticsearch/security"},
-    {"s3": "opendistro-performance-analyzer/opendistro-performance-analyzer", "git": "opendistro-for-elasticsearch/performance-analyzer"},
-    {"s3": "opendistro-index-management/opendistro-index-management", "git": "opendistro-for-elasticsearch/index-management"},
-    {"s3": "opendistro-knn/opendistro-knn", "git": "opendistro-for-elasticsearch/k-NN"},
-    {"s3": "opendistro-knnlib/opendistro-knnlib", "git": "opendistro-for-elasticsearch/k-NN"},
-    {"s3": "opendistro-anomaly-detection/opendistro-anomaly-detection", "git": "opendistro-for-elasticsearch/anomaly-detection"}
+    {
+      "s3":"opendistro-sql/opendistro-sql",
+      "git":"opendistro-for-elasticsearch/sql",
+      "curr_version":"1.9.0.0", "next_version":"1.9.1.0"
```
Comment (weicongs-amazon) [inline bin/plugins.json:80]: I think we should get the versions by checking the release tags. With this way, how should these versions be updated?

Comment (peterzhuamazon) [reply bin/plugins.json:80]: No, these just like release itself, it is fixed / static once it is out.
I would rather hard code them in json than using release tags.
We only create ODFE ES deb/rpm every ODFE release, so individual plugin release does not matter for build.gradle.

Comment (weicongs-amazon) [reply bin/plugins.json:80]: but how and when will it be updated?  In the beginning of the release, we can't know which version will be used for each plugin.

Comment (weicongs-amazon) [reply bin/plugins.json:80]: unless we always use 0.0 for the last 2 digits of plugin version

Comment (peterzhuamazon) [reply bin/plugins.json]: I get it, working on it.

Comment (peterzhuamazon) [reply bin/plugins.json]: Made changes, please check the codes again now.
Thanks.

**Comment by gaiksaya** [comment]
Couple of concerns regarding versions we have been using: 
1. The gradle is using very old version https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/elasticsearch/linux_distributions/gradle/wrapper/gradle-wrapper.properties#L16 can we change it to commonly used among plugin teams? [Example](https://github.com/opendistro-for-elasticsearch/index-management/blob/master/gradle/wrapper/gradle-wrapper.properties#L2)
2. The nebula version is also old that may cause packaging issue with rpm and deb packages https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/elasticsearch/linux_distributions/build.gradle#L25
May be change that too to 8.3.0 (used by plugin teams for building their artifacts) [Example](https://github.com/opendistro-for-elasticsearch/index-management/blob/master/build.gradle#L39)

Also can you put the dependency and names in one line: https://github.com/opendistro-for-elasticsearch/opendistro-build/runs/969252368?check_suite_focus=true#step:5:185 ?

**Comment by peterzhuamazon** [comment]
@gaiksaya I can understand the versions, why need to change to one line for display?
Seems more clear and organized on a new line, especially the plugins are not the same name length.

**Comment by gaiksaya** [comment]
> @gaiksaya I can understand the versions, why need to change to one line for display?
> Seems more clear and organized on a new line, especially the plugins are not the same name length.

Seems confusing to have the connecting info on 2 lines. Like **opendistro-alerting <=1.9.0.0 | > 1.9.0.1** is better than 
**opendistro-alerting
>= 1.9.0.0 | < 1.9.1.0**

**Comment by peterzhuamazon** [comment]
> > @gaiksaya I can understand the versions, why need to change to one line for display?
> > Seems more clear and organized on a new line, especially the plugins are not the same name length.
> 
> Seems confusing to have the connecting info on 2 lines. Like **opendistro-alerting <=1.9.0.0 | > 1.9.0.1** is better than
> opendistro-alerting
> 
> > = 1.9.0.0 | < 1.9.1.0

It's fixed now.

**Pull request merged**